src/C4Config.cpp

```diff
@@ -547,6 +562,34 @@ void C4ConfigGeneral::DeterminePaths(bool forceWorkingDirectory)
 	}
 	else
 		SCopy("/tmp/", TempPath);
+	StdStrBuf xdg_config_path(getenv("XDG_CONFIG_HOME"));
+	if (!xdg_config_path.getSize()){
+	    xdg_config_path = getenv("HOME");
+	    xdg_config_path += "/.config/legacyclonk";
+	}
+	SCopy(xdg_config_path.getData(), XDG_CONFIG_PATH);
+    SCopy(xdg_config_path.getData(), UserPath);
+    SCopy(xdg_config_path.getData(), LogPath); // I'm not sure if this even gets used, but setting it makes the config look nicer :-)
+    StdStrBuf xdg_picture_path(getenv("XDG_PICTURES_DIR"));
+    if (!xdg_picture_path.getSize()){
+        xdg_picture_path = getenv("HOME");
+        xdg_picture_path += "/Pictures";
+    }
+    struct stat st;
+    if (!((stat(xdg_picture_path.getData(), &st) == 0) && S_ISDIR(st.st_mode))) {
+        // fall back to saving screenshots in config folder
+        xdg_picture_path = xdg_config_path;
```

Comment:
That doesn't look right - Screenshots in your config folder? Apart from that being the completely wrong place, there's no subdirectory either.